test/functional/Java9andUp/playlist.xml

```diff
@@ -437,6 +438,33 @@
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
+	</test>
+		<test>
+		<testCaseName>StaticAccessChecks-IllegalAccessDenyWithPermit</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) --illegal-access=permit \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames AccessRestrictedClass \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)
+		</command>
+		<disabled>
+			<comment>https://github.com/eclipse-openj9/openj9/issues/12890</comment>
+		</disabled>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<versions>
+			<version>17+</version>
+		</versions>
+		<impls>
+			<impl>openj9</impl>
+			<impl>ibm</impl>
+		</impls>
```

Comment:
Please use the new format to exclude test. This can be done automatically or manually.
https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#automatically-exclude-a-test-target
https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#manually-exclude-a-test-target

Reply:
Talked with @LongyuZhang , StaticAccessChecks-IllegalAccessPermit is the permanent exclude and StaticAccessChecks-IllegalAccessDenyWithPermit is the temporary exclude, so the change is correct.

Reply:
Talked with @LongyuZhang , StaticAccessChecks-IllegalAccessDenyWithPermit is the permanent exclude and StaticAccessChecks-IllegalAccessDenyWithPermit is temporary exclude, so the change is correct.